rust/mlt-nom/README.md

```diff
@@ -16,3 +16,11 @@ Structure of the data if the `tag` above is 0x01. We should focus this tag on MV
 - each column is defined as:
     - `columnType: varint` - same idea as `tag` above, e.g. `1 = id`, `2 = geometry`, `3 = int property`, etc.
     - TODO...
+
+## Code Structure
+Below, `__NAME__` can be one of these: `FeatureTable`, `Id`, `Geometry` and `Property`. Given the raw input bytes, the parser quickly runs over the input slice and only stores references to the streams and their metadata. Later, decoding can be done on-demand, either for all columns, or just for a specific one.
```

Comment:
nit: Inlining `__NAME__` would imo be much more readable, but likely does not matter